### ISSUE TITLE
eggjs升级2.7.0后egg-oss启动报错

### ISSUE THREAD

**Issue by superlbr** [eggjs/egg#2415]
* **Node Version**: 9.11.1
* **Egg Version**: 2.7.0
* **Plugin Name**: egg-oss
* **Plugin Version**: 1.1.0
* **Platform**: mac
```
/Users/dva/Developer/wapi/node_modules/egg-oss/lib/oss.js:30
  app.createOss = app.oss.createInstance.bind(app.oss);
                          ^
TypeError: Cannot read property 'createInstance' of undefined
    at module.exports.app (/Users/dva/Developer/tuhuo/wapi/node_modules/egg-oss/lib/oss.js:30:27)
    at module.exports.app (/Users/dva/Developer/tuhuo/wapi/node_modules/egg-oss/app.js:6:3)
    at AppWorkerLoader.loadFile (/Users/dva/Developer/tuhuo/wapi/node_modules/egg-core/lib/loader/egg_loader.js:297:30)
```

**Comment by superlbr** [comment]
Another: 
when run yarn autod, egg's version is set back...
Dependencies updates

```
Package Name   Old Version   Latest Version
egg            ^2.7.0        ^2.6.1
```

**Comment by popomore** [comment]
The latest has been changed to 2.6.1. Don’t lock the version.

**Comment by atian25** [comment]
never **lock** the version.

**Comment by ngot** [comment]
never use yarn's lock

**Comment by superlbr** [comment]
@atian25 then use what to make sure all distribution with same correct dependency
Just rollback, is the problem solved?

**Comment by atian25** [comment]
https://zhuanlan.zhihu.com/p/22934066

**Comment by superlbr** [comment]
>『一定要选择靠谱的开源模块』这个前提在实际业务开发中就很难成立

**Comment by superlbr** [comment]
fine. fixed in 2.7.1
